theano/tensor/elemwise.py

```diff
@@ -890,14 +891,15 @@ def perform(self, node, inputs, output_storage):
             # numpy the first (faster) version leads to segfaults
             if self.ufunc:
                 ufunc = self.ufunc
+            elif not hasattr(node.tag, 'ufunc'):
+                # It happen that make_thunk isn't called, like in
+                # get_scalar_constant_value
+                self.prepare_node(node, None, None, 'py')
+                if self.ufunc:
+                    ufunc = self.ufunc
```

Comment:
Fixing get_scalar_constant_value is outside this PR. I'm also not sure we want to force that. I'm not sure how many places would need to be changed to support that.